infra/elasticsearch/iam.tf

```diff
@@ -0,0 +1,43 @@
+data "aws_iam_policy_document" "analytics_logging_role_policy" {
+  "statement" {
+    actions = [
+      "s3:ListBucket",
+      "s3:GetBucketLocation",
+      "s3:ListBucketMultipartUploads",
+      "s3:ListBucketVersions",
+    ]
+
+    effect = "Allow"
+
+    resources = ["${aws_s3_bucket.analytics_logging_bucket.arn}"]
+  }
+
+  statement {
+    actions = [
+      "s3:GetObject",
+      "s3:PutObject",
+      "s3:DeleteObject",
```

Comment:
Does this logging user needs to delete S3 objects?

Reply:
Yes, I want this user to be able to perform delete object on this bucket.  This is the user one of us is running rest calls as on the __kopf__ console.  Think of s3 as a backend rather than a datastore.